pkg/components/prometheus-operator/template.go

```diff
@@ -88,20 +91,28 @@ kubeEtcd:
   enabled: {{.Monitor.Etcd}}
 prometheus-node-exporter:
   service: {}
-{{ if (or .PrometheusOperatorNodeSelector .DisableWebhooks) }}
+{{ if (or .Operator.NodeSelector .DisableWebhooks .Operator.Tolerations ) }}
 prometheusOperator:
   {{- if .DisableWebhooks }}
   tlsProxy:
     enabled: false
```

Comment:
This has been removed a long time ago from upstream: https://github.com/prometheus-community/helm-charts/blob/7fcf3417466f3a3c6c1388167b156b0806a7eb96/charts/kube-prometheus-stack/README.md#from-10x-to-11x

Can we remove it?